ctapipe/image/cleaning.py

```diff
@@ -93,3 +93,42 @@ def dilate(geom, mask):
         input mask (array of booleans) to be dilated
     """
     return mask | geom.neighbor_matrix_sparse.dot(mask)
+
+
+def number_of_islands(geom, mask):
```

Comment:
maybe it would be interesting to also return the clusters, to be able to visualize them